Plots/Satellites/NCL_sat_1.py

```diff
@@ -58,11 +79,16 @@
 ax.add_feature(cfeature.BORDERS, linewidth=.5)
 ax.add_feature(cfeature.LAKES, facecolor='lightcyan', edgecolor='k', linewidth=.5)
 
+# Make array of the contour levels that will be plotted
+contours = np.arange(948, 1060, 4)
+contours = np.append(contours, 975)
```

Comment:
Why are you appending this contour level? You can add a label for the 975 hPa low using a function similar to the one you just defined.

Reply:
I was trying to use levels before and it would throw an error every time I tried to clabel a contour that wasn't in the levels list.

Reply:
Ah understandable. I didn't consider that.